homeassistant/components/amcrest/camera.py

```diff
@@ -144,8 +144,8 @@ async def async_setup_platform(
     registry = entity_registry.async_get(hass)
     old_entity_id = registry.async_get_entity_id(CAMERA_DOMAIN, DOMAIN, serial_number)
     if old_entity_id is not None:
-        _LOGGER.info(f"Updating unique id for camera {old_entity_id}")
-        await hass.async_add_executor_job(entity._update_unique_id)
+        _LOGGER.info("Updating unique id for camera %s", old_entity_id)
```

Comment:
please use f”” as I suggested.
The string used in a function call should be fix not variable.

Reply:
```suggestion
        msg = f"Updating unique id for camera {old_entity_id}”
        _LOGGER.info(msg)
```

Reply:
Thanks for the suggested fix. I was able to roll this into the commit that fixes all of the pylint issues.